public/data/food/ingredients.json

```diff
@@ -3,6 +3,7 @@
     "id": "egg",
     "name": "Oeuf",
     "default": "83752804d1bf53b08b817b1bf9f83d1b",
+    "default_origin": "EuropeAndMaghreb",
```

Comment:
The proper way to edit this file is to edit [ingredients_base.json](https://github.com/MTES-MCT/ecobalyse-data/blob/master/src/ecobalyse_data/food/export_agb/ingredients_base.json) in ecobalyse-data, and have the script `export_builder.py` generate the `ingredients.json` file, and copy it over to ecobalyse.

Reply:
Fixed